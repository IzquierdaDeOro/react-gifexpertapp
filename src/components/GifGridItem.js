import React from 'react'

 const GifGridItem = ({url, title}) => {

    return (
        <div className="card animate__animated animate__bounce">
            <img src={url} alt={title} />
            { title ? <p> {title} </p> : <p> Titulo por defecto </p> }
        </div>
    )
}

export default GifGridItem
