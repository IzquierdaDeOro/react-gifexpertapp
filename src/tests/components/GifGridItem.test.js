import React from 'react';
import {shallow} from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Pruebas en componente GifGridItem', () => {
    
    let wrapper = shallow( <GifGridItem /> );

        beforeEach( () => {

            wrapper = shallow( <GifGridItem /> );

        });

        test('Debe mostrar <GifGridItem /> correctamente', () => {

            expect(wrapper).toMatchSnapshot();

        });
    

})
