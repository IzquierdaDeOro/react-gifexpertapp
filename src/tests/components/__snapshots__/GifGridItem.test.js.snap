// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente GifGridItem Debe mostrar <GifGridItem /> correctamente 1`] = `
<div
  className="card animate__animated animate__bounce"
>
  <img />
  <p>
     Titulo por defecto 
  </p>
</div>
`;
